lib/synapse/config_generator/file_output.rb

```diff
@@ -31,6 +31,8 @@ def tick(watchers)
 
     def update_config(watchers)
       watchers.each do |watcher|
+        next if watcher.config_for_generator[name].nil?
```

Comment:
nit:
should we extra this into a helper method?
we can re-use the method for the other change below as well

```
private

def watch_config_disabled?(watcher)
  watcher.config_for_generator[name] && watcher.config_for_generator[name]['disabled']
end
```